### PR TITLE
Glog 37 make featured carousel as seen in the figma

### PR DESCRIPTION
**Pull request by Chris-Boho**

### CODE REVIEW

**Review by Jhavaa — CHANGES_REQUESTED** [review]
We walked through a bit of it together, so the style is O.K. with me... BUT!... There are some bugs I found:

1. There is an issue with poster not recognizing the cover property, cause some calls are still passing 'image' as an object parameter.
2. The text summary should be left aligned. After looking at it a bit more, I feel like it'll look nicer left-aligned.
3. It would be awesome if you can implement auto sliding to the feature slider. I looked through the docs and found the [autoplay](https://swiperjs.com/swiper-api#autoplay) parameter that does just this. Take a look and see if we can implement that feature.

I'll continue to review the code, so these are just issues observed from running the branch and I _might_ find something else idk. Doubtful tho 😛

File: styles/globals.css [hunk 1]
```diff
@@ -96,14 +96,20 @@ a {
...
 }
 
-.card {
-  /* border: 2px solid black; */
+/* .card {
   width: 300px;
   height: 400px;
+} */
+
+.hope {
```
Comment (Jhavaa) [inline styles/globals.css]: tbh I think we should rename this, cause it's pretty ambiguous lol

File: styles/globals.css [hunk 2]
```diff
@@ -182,7 +200,16 @@ a {
 
 .remove {
   background-color: aliceblue;
+}
+
+.video-responsive {
```
Comment (Jhavaa) [inline styles/globals.css]: I remembered you mentioning that there were multiple of these classes, so we should def consider deleting duplicates.

File: components/FeaturedSlider.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+import "swiper/css";
+import "swiper/css/pagination";
+import "swiper/css/navigation";
+import "swiper/css/scrollbar";
+
+// import required modules
+import { Pagination } from "swiper";
+
+function FeaturedSlider({ gameProp, sliderTitle = "" }) {
+  console.log("---------Game Prop------");
```
Comment (Jhavaa) [inline components/FeaturedSlider.js]: We should avoid merging comments into main, just so we aren't confused by debug console logs in the future. When we start running 'production' level code, we should also avoid any console logs to hide debug code from the user.

File: components/FeaturedSlider.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  console.log(gameProp[1].summary);
+
+  return (
+    <div>
+      <Swiper
+        loop={true}
+        pagination={{
+          dynamicBullets: true,
+        }}
+        style={{
```
Comment (Jhavaa) [inline components/FeaturedSlider.js:29]: I haven't explored swiper that much, but is it possible to put these styles in the CSS? Like import the styles the same way we do with tailwind and define a class with these styles. If not dw about it.

File: components/FeaturedSlider.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                  <div className="divider"></div>
+                  <p className="text-white">
+                    {game.summary
+                      .substring(0, 200)
+                      .substring(
+                        0,
+                        game.summary.substring(0, 200).lastIndexOf(" ")
+                      ) + "..."}
+                    {/* Click the button to watch on Jetflix app. */}
+                    {/* aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa */}
```
Comment (Jhavaa) [inline components/FeaturedSlider.js]: uhhh... no idea what that is 😂 I would say delete comments, but it's up to you if you see yourself going back to this.

**Review by srayce1 — APPROVED** [review]

**Review by Jhavaa — APPROVED** [review]

**Pull request merged**